ankihub/gui/decks_dialog.py

```diff
@@ -706,6 +837,8 @@ def closeEvent(self, event) -> None:
 
 class StudyDeckWithoutHelpButton(StudyDeck):
```

Comment:
:pick: We could rename the class like this, to reflect its purpose better:
```suggestion
class SearchableSelectionDialog(StudyDeck):
```
Could also move it to `ankihub.gui.utils`